httpobs/conf/httpobs.conf

```diff
@@ -31,4 +31,5 @@ database_reconnection_sleep_time = 5
 maintenance_cycle_frequency = 900
 max_load_ratio_per_cpu = 3
 mozilla_domains = mozilla,allizom,browserid,firefox,persona,taskcluster,webmaker
-pinned_domains = accounts.firefox.com,addons.mozilla.org,aus4.mozilla.org,aus5.mozilla.org,cdn.mozilla.org,services.mozilla.com
\ No newline at end of file
+pinned_domains = accounts.firefox.com,addons.mozilla.org,aus4.mozilla.org,aus5.mozilla.org,cdn.mozilla.org,services.mozilla.com
+allow_local = no
```

Comment:
Can you move this up under abort_scan_time?  All of the options are in alphabetical order.